test_acl.py failed due to an import not found

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/Azure/SONiC/wiki#report-issues

---------------------------------------------------
GENERAL SUPPORT INFORMATION
---------------------------------------------------

The GitHub issue tracker is for bug reports and feature requests.
General support can be found at the following locations:

- SONiC Support Forums - https://groups.google.com/forum/#!forum/sonicproject

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Description**

When running test_acl.py, an error: fixture 'mock_server_base_ip_addr' not found

**Steps to reproduce the issue:**
1. run test_acl.py
2.
3.

**Describe the results you received:**
file /var/AzDevOps/sonic-mgmt-master/tests/acl/test_acl.py, line 668
      def test_ingress_unmatched_blocked(self, setup, direction, ptfadapter, ip_version, stage):
file /var/AzDevOps/sonic-mgmt-master/tests/common/fixtures/ptfhost_utils.py, line 227
  @pytest.fixture(scope='module', autouse=True)
  def run_garp_service(duthost, ptfhost, tbinfo, change_mac_addresses, request):
E       fixture 'mock_server_base_ip_addr' not found

**Describe the results you expected:**

PASS
**Additional information you deem important:**
<!--
software/ASIC/Hardware Flatform version and info
-->
    **Output of `show version`:**

    ```

SONiC Software Version: SONiC.master.49439-dirty-20211108.133831
Distribution: Debian 10.11
Kernel: 4.19.0-12-2-amd64
Build commit: a2d0dbe60
Build date: Mon Nov  8 13:52:03 UTC 2021
Built by: AzDevOps@sonic-build-workers-000VEC

Platform: x86_64-ragile_ra-b6510-48v8c-r0
HwSKU: RA-B6510-48V8C
ASIC: broadcom
ASIC Count: 1
Serial Number: www.ruijie.com.cn
Model Number: RJ000001
Hardware Revision: N/A
Uptime: 10:27:48 up 7 min,  1 user,  load average: 1.31, 0.76, 0.37

Docker images:
REPOSITORY                    TAG                                  IMAGE ID            SIZE
docker-dhcp-relay             latest                               ea7ac6fb52f7        435MB
docker-platform-monitor       latest                               fa3bc6a6c6ba        686MB
docker-platform-monitor       master.49439-dirty-20211108.133831   fa3bc6a6c6ba        686MB
docker-snmp                   latest                               ccc75448595e        464MB
docker-snmp                   master.49439-dirty-20211108.133831   ccc75448595e        464MB
docker-teamd                  latest                               62143d9306a6        434MB
docker-teamd                  master.49439-dirty-20211108.133831   62143d9306a6        434MB
docker-router-advertiser      latest                               77522b0dbe09        422MB
docker-router-advertiser      master.49439-dirty-20211108.133831   77522b0dbe09        422MB
docker-nat                    latest                               bbf9549741e9        437MB
docker-nat                    master.49439-dirty-20211108.133831   bbf9549741e9        437MB
docker-lldp                   latest                               301bb342b797        462MB
docker-lldp                   master.49439-dirty-20211108.133831   301bb342b797        462MB
docker-macsec                 latest                               177eab413b9c        437MB
docker-macsec                 master.49439-dirty-20211108.133831   177eab413b9c        437MB
docker-database               latest                               3d9c589afcc9        422MB
docker-database               master.49439-dirty-20211108.133831   3d9c589afcc9        422MB
docker-orchagent              latest                               20034916d841        453MB
docker-orchagent              master.49439-dirty-20211108.133831   20034916d841        453MB
docker-sonic-mgmt-framework   latest                               bc1ec4fd76d3        577MB
docker-sonic-mgmt-framework   master.49439-dirty-20211108.133831   bc1ec4fd76d3        577MB
docker-sonic-telemetry        latest                               57ab18d20b96        510MB
docker-sonic-telemetry        master.49439-dirty-20211108.133831   57ab18d20b96        510MB
docker-mux                    latest                               f6ebeac44bad        474MB
docker-mux                    master.49439-dirty-20211108.133831   f6ebeac44bad        474MB
docker-fpm-frr                latest                               3c9a5b385ad8        452MB
docker-fpm-frr                master.49439-dirty-20211108.133831   3c9a5b385ad8        452MB
docker-sflow                  latest                               1924ea49d7ea        435MB
docker-sflow                  master.49439-dirty-20211108.133831   1924ea49d7ea        435MB
docker-syncd-brcm             latest                               7fa2c8ce5619        738MB
docker-syncd-brcm             master.49439-dirty-20211108.133831   7fa2c8ce5619        738MB
docker-gbsyncd-credo          latest                               aa67d6ad201d        470MB
docker-gbsyncd-credo          master.49439-dirty-20211108.133831   aa67d6ad201d        470MB
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

--- a/tests/acl/test_acl.py
+++ b/tests/acl/test_acl.py
@@ -20,6 +20,7 @@ from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyze
 from tests.common.fixtures.duthost_utils import backup_and_restore_config_db_on_duts
 from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py, run_garp_service, change_mac_addresses
 from tests.common.utilities import wait_until
+from tests.common.dualtor.dual_tor_mock import mock_server_base_ip_addr
 
 logger = logging.getLogger(__name__)
 

